pyjl/transpiler.py

```diff
@@ -231,6 +231,21 @@ def visit_Bytes(self, node):
         bytes_str = "{0}".format(node.s)
         return bytes_str.replace("'", '"')  # replace single quote with double quote
 
+    def visit_Compare(self, node):
+        left = self.visit(node.left)
+        right = self.visit(node.comparators[0])
+
+        comparator_typename = self._typename_from_annotation(node.comparators[0])
+        if comparator_typename and comparator_typename.startswith("Dict"):
```

Comment:
im not keen on these string comparisions.
I see `hasattr(node, "container_type")` elsewhere .. maybe that will work here.

Reply:
yeah - `container_type` is much more robust. Let's go with that.

Reply:
`*generic_container_type` if you're comparing to `Dict`

Reply:
Done. ping @adsharma